keycloak/themes/hmda/login/resources/css/hmda.css

```diff
@@ -163,6 +163,31 @@ body {
   margin-bottom: 0;
 }
 
+#validation_list > li {
+  position: relative;
+}
+
+#validation_list > .complete, #validation_list > .missing {
+  list-style: none;
+}
+
+#validation_list img {
+  display: none;
+  position: absolute;
+  height: 18px;
+  width: 18px;
+  left: -22px;
+  bottom: 5px;
+}
+
+#validation_list > li.complete > .check {
```

Comment:
(See comment above)

~~I think you could just do `#validation_list > li.complete > img` for this rule and the one below it (`#validation_list > li.missing > .missing`). That should take care of both.~~

~~You could also separate them with a `,`, like you've done [above](https://github.com/cfpb/hmda-platform-auth/pull/165/files#diff-e1f62c956c8a0bbd4eeb2efb7debcbafR170).~~

~~Then you could remove the classes on the images, [for example](https://github.com/cfpb/hmda-platform-auth/pull/165/files#diff-0d19d4c8d9acc4f8c775f35627a24d52R43).~~